batch_processing.sh

```diff
@@ -68,6 +68,12 @@ cd t2
 sct_deepseg_sc -i t2.nii.gz -c t2 -qc "$SCT_BP_QC_FOLDER"
 # Tips: If you are not satisfied with the results you can try with another algorithm:
 # sct_propseg -i t2.nii.gz -c t2 -qc "$SCT_BP_QC_FOLDER"
+# Fit binarized centerline from SC seg (default settings)
+sct_get_centerline -i t2_seg.nii.gz -method fitseg -qc "$SCT_BP_QC_FOLDER"
+# Fit binarized centerline from SC seg (trying `-centerline-soft` and `o` flags)
+sct_get_centerline -i t2_seg.nii.gz -method fitseg -centerline-soft 0 -o t2_seg_centerline_bin.nii.gz -qc "$SCT_BP_QC_FOLDER"
```

Comment:
One last nitpick: I don't think we need to include both "default settings" (the first call) _and_  `-centerline-soft 0` (the second call), since they're basically doing the exact same thing? 

So, I think we can remove the `-centerline-soft 0` call:

```suggestion
# Fit binarized centerline from SC seg (default settings)
sct_get_centerline -i t2_seg.nii.gz -method fitseg -qc "$SCT_BP_QC_FOLDER"
```

[`batch_processing.sh`](https://spinalcordtoolbox.com/user_section/getting-started.html#batch-processing-example) is less of an exhaustive test, and is more meant to be a place where we showcase "typical CLI usage" for our users -- and our users won't be specifying `-centerline-soft 0` since that's the default value. :sweat_smile: